Debounce/IDebounceEvents.cs

```diff
@@ -0,0 +1,26 @@
+﻿// SPDX-FileCopyrightText: 2021 Frans van Dorsselaer
+//
+// SPDX-License-Identifier: MIT
+
+namespace Dorssel.Utilities;
+
+/// <summary>
+/// This interface specifies the public API for the <see cref="Debouncer"/> class.
+/// </summary>
+public interface IDebounceEvents : IDebounceSettings
```

Comment:
See comment elsewhere, I don't like the new name. Should be `IDebouncer`.

Reply:
Voortschrijdend inzicht ;)